README.md

```diff
@@ -0,0 +1,389 @@
+  ```
+
+#### Activate profiles:
+By default, ShrinkWrap Resolvers activates profiles based on property value, file presence, active by default profiles, operating system and JDK.
+  ```java
+  Maven.resolver().loadPomFromFile("/path/to/pom.xml", "activate-profile-1", "!disable-profile-2").importRuntimeAndTestDependencies().resolve().withTransitivity().asFile();
+  ```
+
+### Version Range Resolution
+The ShrinkWrap Resolver API allows for resolution of available versions info from a requested range. The [Maven documentation](http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html) specifies the version range syntax.
+```java
+final MavenVersionRangeResult versionRangeResult = Maven.resolver().resolveVersionRange("G:A:[1.0.0]");
+```
+
+### System properties
+
+ShrinkWrap Resolvers allows you to override any programmatic configuration via System properties.
+
+- `org.apache.maven.user-settings`: Path to user settings.xml file, which takes priority in merging over global settings if both are provided.
+
+- `org.apache.maven.global-settings`: Path to global settings.xml file, merged with user settings if provided, with user settings taking priority.
+
+- `settings.security`: Path to settings-security.xml, that contains encrypted master password for password protected Maven repositories.
+
+- `org.apache.maven.offline`: Flag to indicate offline mode.
+
+- `org.apache.maven.flattened-pom-path`: Path to the flattened variant of the regular pom.xml file. Default value is .flattened-pom.xml.
+
+- `maven.repo.local`: Path to local repository with cached artifacts. Overrides value defined in any of the settings.xml files.
+
+- `maven.legacyLocalRepo`: Flag whether to ignore origin tracking for artifacts present in local repository.
+
+
+## Embedded Maven
+
+Embedded Maven allows direct invocation of Maven builds from Java code, offering functionalities such as downloading and using desired Maven binaries, an uncluttered API for simple or complex builds, additional methods for build control (e.g., ignoring failures), and easy access to build outputs and artifacts.
+
+### Downloading Maven Binaries
+
+```java
+// Downloads and uses the specified Maven version, cached in $HOME/.arquillian/resolver/maven/
+EmbeddedMaven.forProject("path/to/pom.xml").useMaven3Version(String version);
+// Uses Maven distribution from the specified URL. 
+// If useCache is false, it downloads to ${project.directory}/target/resolver-maven/downloaded.
+EmbeddedMaven.forProject("path/to/pom.xml").useDistribution(URL mavenDistribution, boolean useCache);
+// Uses Maven installation located on the given path
+EmbeddedMaven.forProject("path/to/pom.xml").useInstallation(File mavenHome);
+// Uses local Maven installation available on your $PATH
+EmbeddedMaven.forProject("path/to/pom.xml").useLocalInstallation();
+// Uses the default Maven version, which is currently 3.3.9
```

Comment:
I'm not so sure about that...

https://github.com/shrinkwrap/resolver/blob/740560792b00dd31306cee7dfff1ba2ec24d1813/maven/api-maven-embedded/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/embedded/DistributionStage.java#L30

Reply:
Probably best to remove the hardcoded version and link the code instead